build-logic-commons/gradle-plugin/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+plugins {
+    `kotlin-dsl`
+}
+
+group = "dev.sigstore.sigstore-gradle"
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
JavaVersion.Version_11?

Reply:
Frankly speaking, I hoped I could support Java 8 and Gradle 6 for plugin.
In other words, the plugin code could produce Java 8 bytecode, and it would spawn Java 11 for actual signing (it would spawn a new Java process anyway for classpath isolation).

However, I've no strong opinion here, and moving to Java 11 might be ok as well

Reply:
I would strongly prefer to be on java 11.